### ISSUE TITLE
Accessibility: Missing Header Skip to Main Content Link

### ISSUE THREAD

**Issue by vkoves** [chihacknight/decarbonize-my-state#91]
Learn more from [this page from University of Illinois](https://fae.disability.illinois.edu/rulesets/BYPASS_1/), but basically we need a skip link so keyboard users don't have to tab through the header after every single navigation.
